src/originalBudgetTrackingApp/export/outputVendors/json/json.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  EnrichedTransaction, ExportTransactionsFunction, OutputVendor, OutputVendorName
+} from '@/originalBudgetTrackingApp/commonTypes';
+import { mergeTransactions, sortByDate } from '@/originalBudgetTrackingApp/transactions/transactions';
+import { promises as fs } from 'fs';
+
+const parseTransactionsFile = async (filename: string) => fs.readFile(filename, { encoding: 'utf8' })
+  .then((content) => JSON.parse(content) as EnrichedTransaction[])
```

Comment:
Why not use async await? The vast majority of the code uses it.

IMO it is clearer syntax.

Reply:
I like the `then` syntax when we have a **pipeline**, you know, you just getting the result of one Promise to another Promise or permutation. I even think in this case it cleaner because the context of each Promise result is just the next `then`.

But of course I need to think again if I'm doing this just in the right places.

Anyway, I converted to `async` `await`. Push soon.

Soory if I have more the ususal English issues, I'm working with the *Github VSCode Extension* so no *Grammarly* at all 😂